toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -670,3 +682,9 @@ func extractWorkerTar(chroot string, workerTar string) (err error) {
 	_, _, err = shell.Execute("tar", "-I", gzipTool, "-xf", workerTar, "-C", chroot)
 	return
 }
+
+func (c *Chroot) GetMountPoints() []*MountPoint {
+	// Create a copy of the list so that the caller can't mess with the list.
```

Comment:
Same ask to document this method. This comment then can be included in that documentation, so the behaviour is discoverable even without entering this file.

Reply:
fixed